src/main/java/sirius/biz/analytics/scheduler/MongoAnalyticalTaskScheduler.java

```diff
@@ -84,15 +85,24 @@ protected boolean isMatchingEntityType(AnalyticalTask<?> task) {
     @Override
     public void executeBatch(JSONObject batchDescription, LocalDate date, int level) {
         if (batchDescription.containsKey(BaseAnalyticalTaskScheduler.CONTEXT_MARKER_GLOBAL_ENTITY)) {
+            AnalyticalEngine.LOG.FINE("Executing global batch: " + batchDescription);
+
             // A global run was detected, execute with "null" and MongoEntity as type...
             executeEntity(null, MongoEntity.class, date, level);
         } else if (batchDescription.containsKey(BaseEntityBatchEmitter.TYPE)) {
+
             // Note that we check for the presence of the TYPE, as (in case of some schedulers) we emit
             // an empty batch, just to ensure that executors with the higher levels are executed, even if
             // there are no tasks on the current level (see AnalyticalBatchExecutor.executeWork...)
-            batchEmitter.evaluateBatch(batchDescription,
-                                       this::extendBatchQuery,
-                                       e -> executeEntity(e, e.getClass(), date, level));
+            AtomicInteger counter = new AtomicInteger();
+            batchEmitter.evaluateBatch(batchDescription, this::extendBatchQuery, e -> {
+                executeEntity(e, e.getClass(), date, level);
+                counter.incrementAndGet();
```

Comment:
What is supposed to happen to the number in the accumulator?

Reply:
I think it should be logged in the message right below?